proto/hummock.proto

```diff
@@ -50,6 +51,8 @@ message IntraLevelDelta {
 
 message GroupConstruct {
   CompactionConfig group_config = 1;
+  uint64 father_group_id = 2;
```

Comment:
Would you mind explaining in description why we need to record `father_group_id`  when split a group?

Reply:
Or in note of code.

Reply:
> Would you mind explaining in description why we need to record `father_group_id` when split a group?

both added.